react/components/Logo/Placeholder.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export default ({ width, height }) => (
```

Comment:
Would not be better if these svgs were on `store-icons`? 🤔

Reply:
@matheusps This PR was just meant to add a padding around the Logo. All these store-components need to be reviewed, and these kind of changes will be done in future PRs